automation/taskcluster/androidTest/flank-x86.yml

```diff
@@ -33,6 +33,9 @@ gcloud:
     - /sdcard/screenshots
   performance-metrics: true
 
+  test-targets:
```

Comment:
Adding this line and below we avoid running these sync integration tests as part as the regular UI tests.